python-lib/google_translate_api_formatting.py

```diff
@@ -92,16 +71,18 @@ def __init__(
         error_handling: ErrorHandlingEnum = ErrorHandlingEnum.LOG,
     ):
         super().__init__(input_df, column_prefix, error_handling)
-        self.translated_text_column_name = "{input_column}_{language}".format(input_column=input_column, language=target_language.replace('-','_'))
+        self.translated_text_column_name = generate_unique(
+            "{input_column}_{language}".format(input_column=input_column, language=target_language.replace('-', '_')),
+            input_df.columns, None)
         self.source_language = source_language
-        if self.translated_text_column_name in input_df:
-            raise Exception("Conflict in column names. {} already exists".format(self.translated_text_column_name))
+        self.input_column = input_column
+        self.target_language = target_language
         self._compute_column_description()
 
     def _compute_column_description(self):
-        self.column_description_dict[
-            self.translated_text_column_name
-        ] = "Input column text translated to the desired language by the google translate api"
+        self.column_description_dict[self.translated_text_column_name] = \
+            "{language} translation of the {col} column by the Google Translation API".format(
+            language=self.target_language, col=self.input_column)
```

Comment:
A few comments here:
- Can we use the human label of the language for the description
- Can we put 'quotes' on {col} (in case there are spaces)?
- As in the description of the plugin, can we use "Google Cloud Translation"?